ios/browser/api/url_sanitizer/BUILD.gn

```diff
@@ -0,0 +1,13 @@
+source_set("url_sanitizer") {
```

Comment:
Missing copyright header

Reply:
I ran `npm run presubmit` and fixed all the header issues